File: MySqlTestInPython/Relation.py
# Base class for Relation.
class Relation(object):
    """description of class"""

    #region Fields

    # The id field of the Relation class.
    __id = None

    # The person1 field of the Relation class.
    __person1 = None

    # The relationship field of the Relation class.
    __relationship = None

    # The person2 field of the Relation class.
    __person2 = None

    # The isDeleted field of the Relation class.
    __isDeleted = None

    #endregion Fields

    #region Properties

    # Gets the id.
    def getId(self):
        return self.__id

    # Sets the id.
    def setId(self, value):
        self.__id = value

    # Gets the person1.
    def getPerson1(self):
        return self.__person1

    # Sets the person1.
    def setPerson1(self, value):
        self.__person1 = value

    # Gets the relationship.
    def getRelationship(self):
        return self.__relationship

    # Sets the relationship.
    def setRelationshipe(self, value):
        self.__relationship = value

    # Gets the person2.
    def getPerson2(self):
        return self.__person2

    # Sets the person2.
    def setPerson2(self, value):
        self.__person2 = value

    # Gets the isDeleted.
    def getIsDeleted(self):
        return self.__isDeleted

    # Sets the isDeleted.
    def setIsDeleted(self, value):
        self.__isDeleted = value

    #endregion Properties

    #region Constructors

    # Initializes a new instance of the Relation class.
    def __init__(self):
        self.__id = None
        self.__person1 = None
        self.__relationship = None
        self.__person2 = None
        self.__isDeleted = None

    # Initializes a new instance of the Relation class.
    def __init__(self, id, person1, relationship, person2, isdeleted):
        self.__id = id
        self.__person1 = person1
        self.__relationship = relationship
        self.__person2 = person2
        self.__isDeleted = isdeleted

    #endregion Constructors

    #region Methods

    #endregion Methods

File: MySqlTestInLua/Relation.lua
-- Base class for Relation.
Relation = {__id = 0, __person1 = 0, __relationship = 0, __person2 = 0, __isDeleted = false}

-- Properties

-- Gets the id.
function Relation : getId()
   return self.__id
end

-- Sets the id.
function Relation : setId(value)
   self.__id = value
end

-- Gets the person1.
function Relation : getPerson1()
   return self.__person1
end

-- Sets the person1.
function Relation : setPerson1(value)
   self.__person1 = value
end

-- Gets the relationship.
function Relation : getRelationship()
   return self.__relationship
end

-- Sets the relationship.
function Relation : setRelationship(value)
   self.__relationship = value
end

-- Gets the person2.
function Relation : getPerson2()
   return self.__person2
end

-- Sets the person2.
function Relation : setPerson2(value)
   self.__person2 = value
end

-- Gets the isDeleted.
function Relation : getIsDeleted()
   return self.__isDeleted
end

-- Sets the isDeleted.
function Relation : setIsDeleted(value)
   self.__isDeleted = value
end

-- Properties

-- Constructors

-- Initializes a new instance of the Relation class.
function Relation : new(o)
   o = o or {}
   setmetatable(o, self)
   self.__index = self
   
   return o
end

-- Initializes a new instance of the Relation class.
function Relation : new(o, id, person1, relationship, person2, isdeleted)
   o = o or {}
   setmetatable(o, self)
   self.__index = self
   self.__id = id or 0
   self.__person1 = person1 or 0
   self.__relationship = relationship or 0
   self.__person2 = person2 or 0
   self.__isDeleted = isdeleted or false
   
   return o
end

-- Constructors